swri_transform_util/include/swri_transform_util/local_xy_util.h

```diff
@@ -80,14 +80,25 @@ namespace swri_transform_util
       double& longitude);
 
   /**
-   * Utility class for converting between WGS84 lat/lon and an ortho-rectified
+   * @brief Utility class for converting between WGS84 lat/lon and an ortho-rectified
```

Comment:
Yep, `@brief` is pointless unless you want to have multiple `@brief` sections that are joined together in the generated documentation.  (and if you do that you've probably missed the point of `@brief`)